about.md

```diff
@@ -1,29 +1,29 @@
 # What is GitLocalize?
 
-[GitLocalize](https://gitlocalize.com) is a continuous localization platform for GitHub repositories and automates the process for developers. Because of the ever-changing nature of Internet-related product, keeping localizing them has been really challenging. GitLocalize solves this by connecting to your GitHub repository and naturally integrating with your development workflow in just a few clicks.
+[GitLocalize](https://gitlocalize.com) is a continuous and automated localization platform that simplifies the workflow process for developers. 
+
+Projects are always changing, and keeping them localized can be a daunting (and annoying) challenge. The makers of GitLocalize realized this and decided that there should be a way to streamline this. GitLocalize connects to your GitHub repository and seamlessly integrates with your development workflow in just a few clicks. 
 
 ## How It Works
 
-You can immediately get started with your GitHub account, choose your repository to integrate with GitLocalize and localize your files. Here are the key features of the platform:
+You can immediately get started with your GitHub account. All you have to do is choose your repository to integrate with GitLocalize and it will localize your files. Here are the key features of the platform:
```

Comment:
Would like to remove `it will localize your files` since to date, GitLocalize itself doesn't localize files